test/dummy/app/controllers/cache_controller.rb

```diff
@@ -1,5 +1,9 @@
 class CacheController < ApplicationController
-  before_filter :instrument_caching
+  Librato::Rails::VersionSpecifier.supported(max: '4.1') do
+    # ActiveSupport::Cache.instrument= was removed in Rails 5.
```

Comment:
You might change this to deprecated in 4.2 so that the version you are limiting on makes sense in context.